spec/plainyearmonth.html

```diff
@@ -269,9 +269,14 @@ <h1>Temporal.PlainYearMonth.prototype.add ( _temporalDurationLike_ [ , _options_
           1. Let _day_ be 1.
         1. Let _date_ be ? CreateTemporalDate(_yearMonth_.[[ISOYear]], _yearMonth_.[[ISOMonth]], _day_, _calendar_).
         1. Let _durationToAdd_ be ? CreateTemporalDuration(_duration_.[[Years]], _duration_.[[Months]], _duration_.[[Weeks]], _balanceResult_.[[Days]], 0, 0, 0, 0, 0, 0).
+        1. Let _optionsCopy_ be ! OrdinaryObjectCreate(%Object.prototype%).
+        1. Let _keys_ be ? EnumerableOwnPropertyNames(_options_, ~key~).
```

Comment:
Why not get both the key and entry here? Then the defining part won’t have to use ? Get.

Reply:
Sounds fine by me, just didn't know that was an option. What's the ecmarkup syntax for iterating over the list returned by `EnumerableOwnPropertyNames(_options_, ~key+value~).`?

Reply:
The same, but you have to manually destructure the entry array